p4-16/spec/P4-16-spec.mdk

```diff
@@ -3766,10 +3804,17 @@ a `struct`---the list fields are assigned to the header fields in the
 order they appear.  In this case the header automatically becomes
 valid:
 
+Similar to a `struct`, a header object can be initialized with a list
+expression [#sec-list-exprs] --- the list fields are assigned to the
+header fields in the order they appear, or with a struct initializer
```

Comment:
Should the comma on this line be replaced with an em dash to close off the dash on the previous line?

"Structure initializer" or "structure initializer expression" instead of "struct initializer"?